app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/projectActionButton.tsx

```diff
@@ -67,69 +73,43 @@ const ProjectActionButton = (props: Props) => {
             return (
                 <>
                     <button onClick={handleOpenProject}>
-                        <heroicons.PlayCircleIcon className="h-10 w-10 text-grey-500" />
+                        <svg width={36} height={36} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="m10.04 7.34 6 3.85a1 1 0 0 1 0 1.68l-6 3.85a1 1 0 0 1-1.54-.84v-7.7a1 1 0 0 1 1.54-.84Z" fill="currentColor" fillOpacity=".8"/>
+                            <rect x={1.5} y={1.5} width={21} height={21} rx={10.5} stroke="currentColor" strokeOpacity={0.1} strokeWidth={3}/>
+                        </svg>
```

Comment:
this is magic. Such things should be exported to variables with nice names of what they define